tests/graph/test_fg.py

```diff
@@ -42,17 +42,23 @@ def test_validate_inputs(self):
         var1 = op1()
         var2 = op2()
 
-        with pytest.raises(TypeError):
+        with pytest.raises(TypeError, match="'Variable' object is not iterable"):
             FunctionGraph(var1, [var2])
 
-        with pytest.raises(TypeError):
+        with pytest.raises(TypeError, match="'Variable' object is not reversible"):
```

Comment:
This one end the above error message is slightly not clear to me. It did not refer to inputs or outputs

Reply:
Feel free to improve the error message